### DISCUSSION TITLE
在 Linux 系统中使用 FHS 标准存储相关文件

### DISCUSSION THREAD

**Discussion by XYenon** [Mrs4s/go-cqhttp#1077] [Ideas]
建议在 Linux 系统中按照 [FHS](https://zh.wikipedia.org/wiki/%E6%96%87%E4%BB%B6%E7%B3%BB%E7%BB%9F%E5%B1%82%E6%AC%A1%E7%BB%93%E6%9E%84%E6%A0%87%E5%87%86) 标准，将 `config.yml`、`device.json` 存入 `/etc/go-cqhttp/`，`data`、`session.token` 存入 `/var/lib/go-cqhtp/`，`logs` 存入 `/var/log/go-cqhttp/`

**Comment by Ink-33** [comment]
FHS标准对于多bot不友好

**Reply by XYenon** [reply]
没有手动指定时，可以按照 `./` `/etc/go-cqhttp/` 的顺序查找配置文件，根据配置文件的位置决定其他文件放在哪，这样对打包和写 systemd service 比较友好

**Reply by wdvxdr1123** [reply]
我觉得有点复杂了，如果需要打包可以使用docker

**Reply by XYenon** [reply]
感觉一个二进制文件做个 image 也挺奇怪的（应该没人在 K8s 之类的里面跑吧
